setup.cfg

```diff
@@ -26,7 +26,7 @@ addopts =
     --doctest-ignore-import-errors --junit-xml=junit-results.xml
     --ignore=doc --ignore=logo --ignore=examples --ignore=tutorials
     --ignore=mne/gui/_*.py --ignore=mne/externals --ignore=mne/icons
-    --capture=sys
+    # --capture=sys
```

Comment:
don't change this.  (it probably won't help anyway, the default is `--capture=fd` which is even more aggresive than `--capture=sys`)

Reply:
Oh yeah sorry, this was scruff to help me test things. The CIs look good!

Reply:
the perils of `git commit -a` come to life!  🙂